vector/src/main/java/im/vector/app/core/extensions/TextView.kt

```diff
@@ -65,12 +75,39 @@ fun TextView.setTextWithColoredPart(@StringRes fullTextRes: Int,
     text = SpannableString(fullText)
             .apply {
                 setSpan(foregroundSpan, index, index + coloredPart.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+                if (onClick != null) {
+                    val clickableSpan = object : ClickableSpan() {
+                        override fun onClick(widget: View) {
+                            onClick()
+                        }
+
+                        override fun updateDrawState(ds: TextPaint) {
+                            ds.color = color
+                            // underline will be handled separately if needed, see below
+                            ds.isUnderlineText = false
```

Comment:
Replace by `ds.isUnderlineText = !underline` to ensure that the clickable part is always underlined?